bin/read-google-sheets-moc-v2.js

```diff
@@ -0,0 +1,74 @@
+const google = require('googleapis');
+const keys = require('../creds-google-sheet.json');
+const googleMethods = require('../server/recess-events/google-methods');
+const firebasedb = require('../server/lib/setupFirebase.js').realtimedb;
+
+const SHEETS_ID = process.env.SHEETS_ID_MOC_EB_IDS;
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets'];
+
+// TODO: change this to actual path
+const ROOT_PATH_MOC = 'test_moc';
+
+// TODO: move this into models script
+class MOC {
+    constructor({ id, name, district, is_incumbent, eventbrite_ids, campaign_eventbrite_ids, is_congress_115, is_congress_116 }) {
+        this.id = id;
+        [this.first_name, this.last_name] = name.trim().split(' ', 2);
+        this.district = district;
+        this.is_incumbent = is_incumbent ? (is_incumbent.toLowerCase() == 'true') : false;
+        this.eventbrite_ids = (eventbrite_ids || '').split(',').map(id => id.trim());
+        this.campaign_eventbrite_ids = (campaign_eventbrite_ids || '').split(',').map(id => id.trim());
+        this.is_congress_115 = is_congress_115 ? (is_congress_115.toLowerCase() == 'true') : false;
+        this.is_congress_116 = is_congress_116 ? (is_congress_116.toLowerCase() == 'true') : false;
+    }
+}
+
+// configure JWT auth client
+let jwtClient = new google.auth.JWT(
+    keys.client_email,
+    null,
+    keys.private_key,
+    SCOPES);
+
+//authenticate request
+jwtClient.authorize(function (err) {
+    if (err) {
+        console.log(err);
+        return;
+    } else {
+        console.log('Successfully connected!');
+    }
+});
+
+// update firebase with google sheets data
+googleMethods.read(jwtClient, SHEETS_ID, 'Sheet1!A:H')
+    .then(data => {
+        let newMocPromises = data
+            // skip the header row
+            .slice(1)
+            // create an new record for each row
+            .map(item => {
+                let moc = new MOC({
+                    id: item[2],
```

Comment:
Could we get the column numbers by looking through the header row? I think a lot of organizers aren't used to the idea that adding or moving columns in a spreadsheet could break the sync for stuff like this.